src/test/shared/defaultAwsClientBuilder.test.ts

```diff
@@ -24,7 +24,7 @@ describe('DefaultAwsClientBuilder', () => {
 
             assert.strictEqual(!!service.config.customUserAgent, true)
             assert.strictEqual(
-                service.config.customUserAgent!,
+                service.config.customUserAgent!.replace('---Insiders', ''),
```

Comment:
Why is this necessary?

Reply:
`ParentProduct` posts `Visual Studio Code - Insiders` instead of `Visual Studio Code` if you're using insiders.

Reply:
Our other toolkits recognize the IDE variants being used. Does it cause an issue?

Reply:
It looks like we also report it in the `Parent Product Version`.

Product: `Visual Studio Code - Insiders`, Version: `1.45.0-insider`

Considering insiders is just a different build, I think we should probably combine them by stripping `- Insiders` from the user agent we post.